website/src/models/group.js

```diff
@@ -475,11 +475,6 @@ GroupSchema.methods.leave = function(user, keep, mainCb){
 
 GroupSchema.methods.toJSON = function() {
   var doc = this.toObject();
-  if(doc.chat){
```

Comment:
It worked for the regular `GET /groups/:id` route, but not for the `GET /groups/:id/chat`